docs/env_files.md

```diff
@@ -0,0 +1,20 @@
+# .env Files
+## Description
+For clients to make calls to the Regulations.gov API, they each need a unique API key. The API key is sensitive information which SHOULD NOT be posted in the GitHub repository. Each client has a corresponding .env file in the `/env_files` directory on the server in this format:
+
```

Comment:
Small edit: put single back quotes around `.env` and other filenames in the document.